examples/telemetry_takeoff_and_land.py

```diff
@@ -6,6 +6,7 @@
 
 
 async def run():
+    """ Arms the system, initiates a takeof and finally lands the system """
     arm_result = await drone.action.arm()
     print("-- Arm result: " + arm_result[1].action_result.result_str)
```

Comment:
I will do something about the `arm_result[1].action_result.result_str`, I don't like to type this and it should be the default output!

Reply:
This PR can be merged, I'll go for a new one!